docs/modules/ROOT/pages/documentation/payara-server/server-configuration/ssl-certificates.adoc

```diff
@@ -66,6 +66,43 @@ In a default installation, the _http-listener-2_ is responsible for handling the
 
 The information can also be entered in the Web Administration console, by navigating to the page Configurations -> server-config -> HTTP Service -> HTTP Listeners > http-listener-2.
 
+[[loading-certificates-from-multiple-keystores]]
+== Loading certificates from multiple keystores
+
+_Since Payara Server 5.2021.5_
+
+IMPORTANT: All additional keystores must have the same password as your domain master password. If you want to load more than one additional keystore use ':' as a delimiter between file paths.
+
+To add additional keystores to the Payara Server you will need to have your keystore and truststore files in a local directory. You can then configure this either in the admin console or using asadmin commands.
+
+The new JVM properties used to add additional keystores are:
+
+* -Dfish.payara.ssl.additionalKeyStores
+* -Dfish.payara.ssl.additionalTrustStores
+
+=== Using the Admin Console
+To configure the additional keystore locations in the admin console, head to the *Configurations -> <instance configuration> -> JVM Settings* and on the *JVM Options* tab click *Add JVM Option*.
+
+In the new blank JVM property, you can add the new JVM properties and the relative paths to your keystores and truststores respectively.
```

Comment:
`new blank JVM propert**y**` ... `new JVM propert**ies**`
The grammar pedant in me cries